miden/tests/integration/stdlib/crypto/ecdsa_secp256k1.rs

```diff
@@ -0,0 +1,58 @@
+use super::build_test;
+use test_case::test_case;
+
+// Wrapper types for ease of writing parameterized test cases
+struct FieldElement([u32; 8]);
+
+type BaseField = FieldElement;
+type ScalarField = FieldElement;
+
+struct Point([BaseField; 3]);
+
+#[test_case(Point([FieldElement([1187647059, 1135132293, 1524607722, 3257770169, 1812770566, 4163599075, 3343690625, 2983146250]), FieldElement([694970425, 3961647168, 2962892522, 3871680339, 479244527, 2106589630, 3531004100, 487738481]), FieldElement([1718928786, 2222219308, 1537333708, 969814285, 1600645591, 2744076726, 1359599981, 1095895041])]), FieldElement([1915140291, 1682821516, 1088031394, 2866424576, 2852209138, 1159876682, 234168247, 3360002988]), FieldElement([1494159694, 3668493121, 2315165624, 353127114, 974571799, 2051320959, 3421809437, 3258836281]), FieldElement([1259054195, 60155476, 2236955964, 2106542718, 1332177784, 1407189293, 11489664, 3695133146]) ; "0")]
+#[ignore]
```

Comment:
Can we leave a comment on why this test is ignored?

Reply:
Added in https://github.com/maticnetwork/miden/pull/489/commits/a2c90a895ebc8789ca6d2cd70f95a6ffa9fb7f5a